plugins/org/src/components/Cards/Group/GroupProfile/GroupProfileCard.tsx

```diff
@@ -80,6 +85,18 @@ export const GroupProfileCard = ({
       title={<CardTitle title={displayName} />}
       subheader={description}
       variant={variant}
+      action={
+        <IconButton
+          aria-label="Edit"
+          disabled={!entityMetadataEditUrl}
+          title="Edit Metadata"
+          onClick={() => {
+            window.open(entityMetadataEditUrl ?? '#', '_blank');
```

Comment:
not a fan of javascript for this - can you make use of the `Link` component in `core-components`  instead which already has handling for how to deal with links and opening in new window if it's an external link etc etc? I think you should be able to do `<IconButton component={Link}` to get the link behavior but the icon button looks

Reply:
Okay will update it

Reply:
Hi @freben I don't think IconButton has a component prop
https://material-ui.com/api/icon-button/

Reply:
`Any other props supplied will be provided to the root element ([ButtonBase](https://material-ui.com/api/button-base/)).`

Reply:
oops, my bad, didn't see until the bottom, thanks

Reply:
already push the changes @freben , maybe you could help to review it again, maybe we should change another page too to use Link